package com.example.demo.repository;

import com.example.demo.model.User;
import java.util.List;

public interface UserRepository {
    List<User> findAll();
    User save(User user);
}
